src/lib/nnlib.jl

```diff
@@ -1,12 +1,23 @@
 using NNlib
 import NNlib: softmax, ∇softmax, logsoftmax, ∇logsoftmax, conv, depthwiseconv, ∇conv_data, ∇depthwiseconv_data, maxpool, meanpool, σ, relu, batched_mul, batched_adjoint
+import LoopVectorization: vifelse
```

Comment:
Based on current discussion, vectorisation would need some changes to how we plan to integrate lv within lv. Could we remove the dep from this PR?

Reply:
Oh, my bad, sure, I'll return it.